libtiledbsoma/src/soma_array_reader.cc

```diff
@@ -125,7 +125,8 @@ void SOMAArrayReader::reset(
     batch_size_ = batch_size;
 
     result_order_ = "auto";
-    if (result_order != "auto") {  // default "auto" is set in soma_reader.h
+    if (result_order !=
+        "auto") {  // default "auto" is set in soma_array_reader.h
```

Comment:
(I understand it's all automatic formatting and what not but I really do think line 128 was more readable than the entirely equiivalent pair of 128+129 which blends it into the indented block.  Anyway, that ship has sailed.)

Reply:
@eddelbuettel it has, and the PR CI failed until/unless I ran `make format`.